src/02-room-details.js

```diff
@@ -49,7 +64,23 @@ function getRoomByDinosaurName(dinosaurs, rooms, dinosaurName) {}
       "Kit Hopkins Education Wing"
     ]
  */
-function getConnectedRoomNamesById(rooms, id) {}
+const getConnectedRoomNamesById = (rooms, id) => {
+  const connectedRoom = rooms.find(room => room.roomId === id);
+  if (!connectedRoom) {
+    return `Room with ID of '${id}' could not be found.`;
+  }
+  
+  const connectedRoomIds = connectedRoom.connectsTo;
+  const connectedRoomNames = rooms.filter(room => connectedRoomIds.includes(room.roomId)).map(room => room.name);
```

Comment:
Can we refactor this chaining of operations?  The data is hard to follow but I think there may be a way to make this easier to read even if it is very short